Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs

```diff
@@ -157,21 +157,45 @@ public QuestionRepository(TrappistDbContext dbContext)
 
         public async Task UpdateSingleMultipleAnswerQuestionAsync(int questionId, QuestionAC questionAC, string userId)
         {
-            var updatedQuestion = Mapper.Map<QuestionDetailAC, Question>(questionAC.Question);
-            var options = questionAC.SingleMultipleAnswerQuestion.SingleMultipleAnswerQuestionOption;
+            var question = new Question();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused 'question' local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)